docs/hub/datasets-duckdb-select.md

```diff
@@ -0,0 +1,164 @@
+# Query datasets
+
+Querying datasets is a fundamental step in data analysis. Here, we'll guide you through querying datasets using various methods.
+
+There are [several ways](https://duckdb.org/docs/data/parquet/overview.html) to select your data.
+
+Using the `FROM` syntax:
+```bash
+FROM 'hf://datasets/jamescalam/world-cities-geo/train.jsonl' SELECT city, country, region LIMIT 3;
+
+┌────────────────┬─────────────┬───────────────┐
+│      city      │   country   │    region     │
+│    varchar     │   varchar   │    varchar    │
+├────────────────┼─────────────┼───────────────┤
+│ Kabul          │ Afghanistan │ Southern Asia │
+│ Kandahar       │ Afghanistan │ Southern Asia │
+│ Mazar-e Sharif │ Afghanistan │ Southern Asia │
+└────────────────┴─────────────┴───────────────┘
+
+```
+
+Using the `SELECT` and `FROM` syntax:
+
+```bash
+SELECT city, country, region FROM 'hf://datasets/jamescalam/world-cities-geo/train.jsonl' USING SAMPLE 3;
+
+┌──────────┬─────────┬────────────────┐
+│   city   │ country │     region     │
+│ varchar  │ varchar │    varchar     │
+├──────────┼─────────┼────────────────┤
+│ Wenzhou  │ China   │ Eastern Asia   │
+│ Valdez   │ Ecuador │ South America  │
+│ Aplahoue │ Benin   │ Western Africa │
+└──────────┴─────────┴────────────────┘
+
+```
+
+Count all JSONL files matching a glob pattern:
+
+```bash
+SELECT COUNT(*) FROM 'hf://datasets/jamescalam/world-cities-geo/*.jsonl';
+
+┌──────────────┐
+│ count_star() │
+│    int64     │
+├──────────────┤
+│         9083 │
+└──────────────┘
+
+```
+
+You can also query Parquet files using the `read_parquet` function (or its alias `parquet_scan`). Let's explore these functions using the auto-converted Parquet files from the same dataset.
```

Comment:
maybe mention that `read_parquet` is actually not required? because later on there are examples where it is skipped without explanation

Reply:
I added information to highlight that the functions are more useful when the files don't end with the `.parquet` extension.